### PR TITLE
Elaborate on Coordinator quote verification 

### PR DESCRIPTION
**Pull request by Nirusu**
Another chunk of https://github.com/edgelesssys/marblerun/issues/213

### CODE REVIEW

File: marblerun/workflows/verification.md [hunk 1]
```diff
@@ -6,33 +6,33 @@ MarbleRun provides a simple HTTP REST API for clients to verify the confidential
...
-For testing, we have published a Coordinator image at `ghcr.io/edgelesssys/coordinator`.
+In production, the expected values in `coordinator-era.json` would be generated when building the Coordinator and distributed to your clients. When you build MarbleRun from source, you can find the file in your build directory.
+For testing with a pre-built release, we have published a Coordinator image at `ghcr.io/edgelesssys/coordinator`.
 You can pull the corresponding `coordinator-era.json` file from our release page:
 
 ```bash
 wget https://github.com/edgelesssys/marblerun/releases/latest/download/coordinator-era.json
 ```
 
+After installing Era, you can verify the quote with the following command:
```
Comment (m1ghtym0) [inline marblerun/workflows/verification.md:21]:
```suggestion
After installing era, you can verify the quote with the following command:
```

**Review by m1ghtym0 — APPROVED** [review]
Thanks!

File: marblerun/workflows/verification.md (hunk 1 above)
Comment (Nirusu) [reply marblerun/workflows/verification.md:21]: It's capitalized before in the text. Change that one, too?

Comment (m1ghtym0) [reply marblerun/workflows/verification.md:21]: Oh, I've missed that. Yes please.

**Review by m1ghtym0 — APPROVED** [review]

**Pull request merged**